tests/test_pipelines_object_detection.py

```diff
@@ -46,16 +48,22 @@ def open(*args, **kwargs):
             pass
 
 
+@require_detectron2
 @require_vision
 @require_timm
 @require_torch
+@require_pytesseract
 @is_pipeline_test
 class ObjectDetectionPipelineTests(unittest.TestCase, metaclass=PipelineTestCaseMeta):
     model_mapping = MODEL_FOR_OBJECT_DETECTION_MAPPING
 
+    @require_detectron2
     @require_datasets
+    @require_pytesseract
     def run_pipeline_test(self, model, tokenizer, feature_extractor):
-        object_detector = ObjectDetectionPipeline(model=model, feature_extractor=feature_extractor)
+        object_detector = ObjectDetectionPipeline(
+            model=model, tokenizer=tokenizer, feature_extractor=feature_extractor
+        )
         outputs = object_detector("./tests/fixtures/tests_samples/COCO/000000039769.png", threshold=0.0)
```

Comment:
Adding `detectron2` here means that previous object detection models (DETR) will not be run. The CircleCI run that has the `detectron2` dependency only runs the LayoutLM-v2 tests.

Reply:
@Narsil will address this comment and will merge

Reply:
@LysandreJik would removing `require_detectron2` & `require_pytesseract` decorators from  `run_pipeline_test` work? (since these decorators are being required at a class level)
https://github.com/huggingface/transformers/blob/e6107347b67537f807cb0f2c4321118da5278271/tests/test_pipelines_object_detection.py#L60-L62

https://github.com/huggingface/transformers/blob/e6107347b67537f807cb0f2c4321118da5278271/tests/test_pipelines_object_detection.py#L51-L57

If not, please let me know what would be the best way to handle this problem